packages/playground/Samples/accessible.tsx

```diff
@@ -117,6 +117,22 @@ export default class Bootstrap extends React.Component<
           <Text style={styles.text}>TEST setAccessibilityFocus</Text>
         </TouchableHighlight>
         <TextInput ref={this.myElement} />
+
+        <TouchableHighlight
```

Comment:
We typically leave these sample pages as they are. If you want to add an example of the prop, add an example to a JS/TS file in the rntester app (i.e. the files that have the *Example syntax). I would add an example to AccessibilityExampleWindows.tsx